src/main/scala/mesosphere/chaos/http/impl/HttpServer.scala

```diff
@@ -151,15 +161,6 @@ class HttpServer(conf: HttpConf, registry: MetricRegistry) extends ServletServer
 
   private lazy val requestLog: RequestLog = new ChaosRequestLog
 
-  private lazy val resourceHandler: ResourceHandler = {
```

Comment:
The resourceHandler was used to serve resource assets. 
We do not use this functionality any longer.